lib/theme/view.js

```diff
@@ -61,7 +67,9 @@ View.prototype.renderSync = function(options = {}) {
 };
 
 View.prototype._buildLocals = function(locals) {
-  return assignIn({}, locals, omit(this.data, ['layout', '_content']), {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
can use the following function (which re-implement lodash.omit) to avoid unused vars ([source](https://github.com/Semantic-Org/Semantic-UI-React/pull/860)),

``` js
Object.keys(props).reduce((acc, prop) => {
    if (handledProps.indexOf(prop) === -1) acc[prop] = props[prop]
    return acc
  }, {})
```